.github/workflows/deploy.yml

```diff
@@ -74,7 +74,7 @@ jobs:
 
   deploy:
     runs-on: ubuntu-latest
-    needs: [detect-environments, acceptance-test]
+    needs: [detect-environments]
```

Comment:
are we ok not mandating the acceptance tests to pass for a deploy? ideally we want to be able to deploy to dev without needing these to pass

Reply:
Deploy is a misnomer for this pipeline; really it should say Publish.  I guess the nuance is that publishing an image does currently result in an automatic deploy via CodeBuild in AWS.  However, I still think the pipeline should not say deploy, since it doesn't directly do that.  As for the tests, the ideal scenario is that we do require them to pass since we'd like to avoid publishing a defective image.  For now though, while we're getting the tests settled, I'd be comfortable about not requiring them.